py/server/tests/test_parquet.py

```diff
@@ -347,7 +347,7 @@ def test_dates_and_time(self):
         from_disk = read('data_from_dh.parquet')
         self.assert_table_equals(dh_table, from_disk)
 
-        df_from_disk = to_pandas(from_disk)
+        df_from_disk = to_pandas(from_disk, dtype_backend=None)
         if pandas.__version__.split('.')[0] == "1":
             df_from_pandas = pandas.read_parquet("data_from_dh.parquet", use_nullable_dtypes=True)
         else:
```

Comment:
This looks suspect.  We are doing our on data treatment with `dtype_backend=None`, and then we read with `use_nullable_dtypes=True`.

Reply:
Debugged a bit with @malhotrashivam and we found an issue with `to_arrow`. https://github.com/deephaven/deephaven-core/issues/4823